### DISCUSSION TITLE
Win 11 issue

### DISCUSSION THREAD

**Discussion by mpeline** [fernandreu/office-ribbonx-editor#246] [Q&A]
I made a custom tab for Excel with buttons to run some Macros, and it works fine under Windows 7 - 10.
However, when running the same file under Win 11 it takes forever to load the custom tab.
The Excel file is stored on a shared drive, and only users running Win 11 have issues.
We are running Office 360
Any suggestions how to resolve this problem?

**Comment by fernandreu** [comment]
Sorry, this is not a problem with the tool itself so I cannot help. I will move this to the discussions section in case somebody wants to do so there (assuming you didn't find a way around it by now... it's been a while)
